Note: `go test ./...` currently fails on clean `origin/main` at
`19575d63118cf95564dd4755d490367be13caa1a` before this PR was applied.

The failures are in client/client-config/thin, mostly:

   ` transport: no listen transport configured`

and one thin-client fixture failure:

    `open testdata/thinclient.toml: no such file or directory`

The code touched by this PR have passing tests on top of main:

```
CGO_LDFLAGS="-luring -lbz2" go test -mod=readonly ./authority/voting/... ./replica/... -count=1
ok  	github.com/katzenpost/katzenpost/authority/voting/client	18.563s
ok  	github.com/katzenpost/katzenpost/authority/voting/server	0.327s
ok  	github.com/katzenpost/katzenpost/authority/voting/server/config	0.003s
?   	github.com/katzenpost/katzenpost/authority/voting/server/profiling	[no test files]
ok  	github.com/katzenpost/katzenpost/replica	7.600s
ok  	github.com/katzenpost/katzenpost/replica/common	1.661s
ok  	github.com/katzenpost/katzenpost/replica/config	0.003s
ok  	github.com/katzenpost/katzenpost/replica/cryptography_model_tests	10.584s
?   	github.com/katzenpost/katzenpost/replica/instrument	[no test files]
ok  	github.com/katzenpost/katzenpost/replica/integration_tests	126.504s
```

Unfortunately the current full-suite failure is happening and it appears unrelated to this PR as main and the latest tagged version seem to be failing:
```
git checkout main
git reset --hard origin/main
git status
go version
go env GOTOOLCHAIN GOFLAGS GOOS GOARCH
git grep -n "no listen transport configured"
git ls-tree -r HEAD client/thin/testdata
CGO_LDFLAGS="-luring -lbz2" go test -mod=readonly ./client/config -run TestConfig -count=1 -v
CGO_LDFLAGS="-luring -lbz2" go test -mod=readonly ./client/thin -run TestLoadFile -count=1 -v
```

The commands above show this result:
```
Already on 'main'
Your branch is up to date with 'origin/main'.
HEAD is now at 19575d631 Use latest hpqc, for more post quantum signature schemes (#1005)
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
go version go1.26.2 linux/amd64
auto

linux
amd64
client/transport/transport.go:35:var ErrNoTransport = errors.New("transport: no listen transport configured")
120000 blob 4d58be183f57942e6b10dc0998d75f36b1d996be	client/thin/testdata/thinclient.toml
=== RUN   TestConfig
    config_test.go:11: 
        	Error Trace:	/home/user/Documents/src/katzenpost/client/config/config_test.go:11
        	Error:      	Received unexpected error:
        	            	transport: no listen transport configured
        	Test:       	TestConfig
--- FAIL: TestConfig (0.00s)
FAIL
FAIL	github.com/katzenpost/katzenpost/client/config	0.004s
FAIL
=== RUN   TestLoadFile
    thin_unit_test.go:95: 
        	Error Trace:	/home/user/Documents/src/katzenpost/client/thin/thin_unit_test.go:95
        	Error:      	Received unexpected error:
        	            	open testdata/thinclient.toml: no such file or directory
        	Test:       	TestLoadFile
--- FAIL: TestLoadFile (0.00s)
=== RUN   TestLoadFileNonexistent
--- PASS: TestLoadFileNonexistent (0.00s)
=== RUN   TestLoadFileInvalidTOML
--- PASS: TestLoadFileInvalidTOML (0.00s)
=== RUN   TestLoadFile_RejectsOldFlatFormat
--- PASS: TestLoadFile_RejectsOldFlatFormat (0.00s)
=== RUN   TestLoadFile_RejectsUnknownTopLevelKey
--- PASS: TestLoadFile_RejectsUnknownTopLevelKey (0.00s)
=== RUN   TestLoadFile_RejectsMissingPigeonholeGeometry
--- PASS: TestLoadFile_RejectsMissingPigeonholeGeometry (0.00s)
=== RUN   TestLoadFile_RejectsMissingDial
--- PASS: TestLoadFile_RejectsMissingDial (0.00s)
=== RUN   TestLoadFile_RejectsUnknownDialSubtable
--- PASS: TestLoadFile_RejectsUnknownDialSubtable (0.00s)
=== RUN   TestLoadFile_RejectsUnknownPigeonholeGeometryKey
--- PASS: TestLoadFile_RejectsUnknownPigeonholeGeometryKey (0.00s)
FAIL
FAIL	github.com/katzenpost/katzenpost/client/thin	0.003s
FAIL
```

I also ran the above on a ppc64le (production) system and had the same problems.

--- a/authority/voting/client/client.go
+++ b/authority/voting/client/client.go
@@ -488,8 +488,11 @@ func descriptorPostRetryDelay(cfg *Config, attempts int) time.Duration {
 	}
 
 	maxDelay := cfg.RetryMaxDelay
-	if maxDelay < base {
+	if maxDelay < 20*time.Second {
 		maxDelay = 20 * time.Second
+	}
+	if maxDelay < base {
+		maxDelay = base
 	}
 
 	delay := base

